docs/Guides/Ecosystem.md

```diff
@@ -341,6 +341,8 @@ section.
   authentication for Fastify-based web apps.
 - [`fastify-kafkajs`](https://github.com/kffl/fastify-kafkajs) Fastify plugin
   that adds support for KafkaJS - a modern Apache Kafka client library.
+- [`fastify-keycloak-adapter`](https://github.com/yubinTW/fastify-keycloak-adapter) 
```

Comment:
Is there any whitespace at the end of this line? We use https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md013---line-length with `strict: false` and `stern: false`. So this line _should_ be okay if there is not any whitespace at the end of the line.

Reply:
It's my fault, I will remove the whitespace.

Reply:
I have fixed it, please review :)